bundle/manifests/rhdh.redhat.com_orchestrators.yaml

```diff
@@ -412,54 +380,30 @@ spec:
                     description: Orchestrator plugin information
                     properties:
                       integrity:
-                        default: sha512-A/twx1SOOGDQjglLzOxQikKO0XOdPP1jh2lj9Y/92bLox8mT+eaZpub8YLwR2mb7LsUIUImg+U6VnKwoAV9ATA==
+                        default: sha512-vGGd9hUmDriEMmP2TfzLVa3JSnfot2Blg+aftDnu/lEphsY1s2gdA4Z5lCxUk7aobcKE6JO/f0sIl2jyxZ7ktw==
                         description: Package SHA integrity
                         type: string
                       package:
-                        default: backstage-plugin-orchestrator@1.3.0
+                        default: backstage-plugin-orchestrator@1.4.0-rc.3
                         description: Package name
                         type: string
                     type: object
                   orchestratorBackend:
                     description: Orchestrator backend plugin information
                     properties:
                       integrity:
-                        default: sha512-Th5vmwyhHyhURwQo28++PPHTvxGSFScSHPJyofIdE5gTAb87ncyfyBkipSDq7fwj4L8CQTXa4YP6A2EkHW1npg==
+                        default: sha512-tS5cJGwjzP9esdTZvUFjw0O7+w9gGBI/+VvJrtqYJBDGXcEAq9iYixGk67ddQVW5eeUM7Tk1WqJlNj282aAWww==
                         description: Package SHA integrity
                         type: string
                       package:
-                        default: backstage-plugin-orchestrator-backend-dynamic@1.3.0
+                        default: backstage-plugin-orchestrator-backend-dynamic@1.4.0-rc.3
```

Comment:
I'd suggest to split the PR into 2:
* One PR that upgrades the CRD spec to the v1alpha2 with the changes we agreed (removing the plugins from the spec and changes requested by @jianrongzhang89 )
* One PR that updates the plugin versions in preparation for a new release.

Reply:
I thought @jianrongzhang89 already has a PR for this changes and would rebase after we merged this. I didn't know if he wanted those changes in this one. Will sync up with him now.

Yeah at the beginning, it was supposed to be just plugin changes and do rc-1 release afterwards. In hindsight, I should have split it into two PRs

Reply:
I thought @jianrongzhang89 already has a PR for this changes and would rebase after we merged this. I didn't know if he wanted those changes in this one. Will sync up with him now.

Yeah at the beginning, it was supposed to be just plugin changes and do rc-1 release afterwards. In hindsight, I should have split it into two PRs

Reply:
Updated - removed the new plugin version changes